pyscriptjs/src/plugins/python/py_tutor.py

```diff
@@ -0,0 +1,202 @@
+import html
+
+from pyscript import Plugin, js
+
+js.console.warn(
+    "WARNING: This plugin is still in a very experimental phase and will likely change"
+    " and potentially break in the future releases. Use it with caution."
+)
+
+plugin = Plugin("PyTutorial")
+
+# TODO: Part of the CSS is hidden in examples.css ---->> IMPORTANT: move it here!!
+
+# TODO: Python files running and <py-script src="bla.py"> not in the config are not available...
```

Comment:
So we have to explicitly fetch the file?

Reply:
Yes, for now we do... But I want to open an issue for this. Just didn't want to make this a huge and long PR (it's already taken more time and code than I expected :) )

Reply:
haha yeah that makes sense 😄